src/emulator.ts

```diff
@@ -311,6 +332,13 @@ export class EmacsEmulator implements IEmacsController, vscode.Disposable {
     }
   }
 
+  /**
```

Comment:
Such documentation is trivial as it just explains what we can understand easily just by reading inside the callback.
Add meaningful docs only.